app/jobs/catalogue_check_job.rb

```diff
@@ -0,0 +1,76 @@
+class CatalogueCheckWorker < ActiveJob::Base
+  queue_as :default
+
+  require 'capybara'
+  require 'capybara/poltergeist'
+  require_relative '../../app/jobs/scrape_job.rb'
+  require_relative '../../app/jobs/delete_job.rb'
+
+  include Capybara::DSL
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app, timeout: 60, js_errors: false)
+  end
+  Capybara.default_driver = :poltergeist
+
+  def perform
+
+    #setting the initial variable
+    current_catalogue = Hash.new(0)

```

Comment:
The conventional way to initialize an empty `Hash` is to use the hash literal `{}`:

```ruby
# Prefer {} to Hash.new
current_catalogue = {}
```
I think that'll work unless there's a good reason for the current_catalogue to have a default of `0`. Consider reading http://ruby-doc.org/core-2.2.0/Hash.html#method-c-new for more info on initializing Hashes with default values and blocks.